### ISSUE TITLE
Account details page flashes every time you sign in

### ISSUE THREAD

**Issue by lukemeierj** [jack-and-the-beans/buddies-ios#135]
When you sign in, the bio/name/profile page appears briefly before showing you Discover.

**Comment by lukemeierj** [comment]
I'm guessing we won't fix this.  Thoughts?

**Comment by lukemeierj** [comment]
It sometimes lingers for fairly long.  Like a whole second

**Comment by JakeThurman** [comment]
I fixed this at one point. It must have gotten added back in Noah's changes, unfortunately.

I'm guessing we're now calling back to `useLoggedInUser` with `nil` if it's not loaded yet. This is what causes this flash. The intent was to wait on the splash screen. 

Unless I'm misunderstanding and this is truly "Sign In" *only* rather than "App Open" (in addition to on sign in). In which case, I don't have any leads other than it is likely related.

**Comment by lukemeierj** [comment]
I don't recall it on "App open", but certainly on sign in.

**Comment by noahtallen** [comment]
@lukemeierj I do remember you reported a similar bug where it happened every time you tried to open the app - that's definitely fixed now. I am also not seeing this happen when logging in with email.

**Comment by noahtallen** [comment]
In fact, on `master`, I cannot reproduce this bug on either a real device or emulator. I've tried logging in with facebook, with email, just opening the app, and all seem fine.

If this is still happening for you guys and you're based on master, feel free to re-open
